lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -666,7 +663,11 @@ function dosomething_global_node_url($nid, $language, $fragment = NULL) {
   if (isset($fragment)) {
     $raw_url .= '#' . $fragment;
   }
-  return $GLOBALS['base_url'] . '/' . dosomething_global_get_prefix_for_language($language) . '/' . drupal_get_path_alias($raw_url, $language);
+  $alias = dosomething_global_get_prefix_for_language($language);
```

Comment:
Please rename this variable, `$alias` is incorrect name and very confusing. This should be named `$prefix`.
Actual alias is returned by `drupal_get_path_alias()`.